src/tests/functional/plugin/cpu/single_layer_tests/activation.cpp

```diff
@@ -168,17 +168,17 @@ std::vector<std::vector<ov::Shape>> basic3D = {
         {{2, 17, 5}},
 };
 
-const auto basicCases3D = ::testing::Combine(
+const auto basicCases3D_Eltwise = ::testing::Combine(
         ::testing::ValuesIn(static_shapes_to_test_representation(basic3D)),
         ::testing::Values(activationShapes),
         ::testing::ValuesIn(CommonTestUtils::combineParams(activationTypes)),
-        ::testing::ValuesIn(netPrc),
+        ::testing::Values(Precision::FP32),  // need to support nCw[8/16]c, nwc for Snippets
```

Comment:
Minor: I didn't get how layout is connected with precision, and why this comment is here. move/remove/elaborate?